test/mongooseim_metrics_SUITE.erl

```diff
@@ -11,24 +11,26 @@
 all() ->
     [
      {group, ordinary_mode},
-     {group, all_metrics_are_global}
+     {group, all_metrics_are_global},
+     {group, all_metrics}
     ].
 
 -define(ALL_CASES, [no_skip_metric, subscriptions_initialised]).
 
 groups() ->
     [
      {ordinary_mode, [], ?ALL_CASES},
-     {all_metrics_are_global, [], ?ALL_CASES}
+     {all_metrics_are_global, [], ?ALL_CASES},
+     {all_metrics, [parallel], all_metrics_list()}
```

Comment:
What is the purpose of this new group? It definitely doesn't verify "all metrics".

Reply:
Hmmm... I made a new group to try out my new tests and later on didn't really remember to give it a more in depth thought, good reminder. I think I can just put both new tests on both old groups, right?

Reply:
Both metrics are verified in big tests. No reason to put them here as well.

Reply:
I thought it was good to have some small tests as well, in contrast to everything being tested in the big ones alone.